pykickstart/commands/autopart.py

```diff
@@ -598,6 +598,69 @@ def parse(self, args):
 
         return retval
 
+
+class F41_AutoPart(F38_AutoPart):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.reuse = kwargs.get("reuse", [])
+        self.erase = kwargs.get("erase", [])
+        self.remove = kwargs.get("remove", [])
+
+    def __str__(self):
+        retval = super().__str__()
+        if not self.autopart:
+            return retval
+
+        if retval and self.reuse:
+            retval = retval.rstrip()
+            retval += " --reuse=" + ",".join(self.reuse)
+        if retval and self.erase:
+            retval = retval.rstrip()
+            retval += " --erase=" + ",".join(self.erase)
+        if retval and self.remove:
+            retval = retval.rstrip()
+            retval += " --remove=" + ",".join(self.remove)
+
+            retval += "\n"
+        return retval
+
+    def _getParser(self):
+        op = super()._getParser()
+        op.add_argument("--reuse", dest="reuse", type=commaSplit,
+                        version=F41, help="""
+                        Specifies which existing mountpoins should be
+                        reused.
+
+                        Single device must exist for given mountpoint.
+
+                        ``autopart --reuse=/home``""")
+        op.add_argument("--erase", dest="erase", type=commaSplit,
+                        version=F41, help="""
+                        Specifies which existing mountpoins should be
```

Comment:
I would assume this is non-recursive, right? May want to make a note of that just to be clear that erasing / doesn't also erase /home if it is a separate mountpoint.